$OpenBSD: patch-_metadata,v 1.1 2015/07/02 11:27:24 jasper Exp $

Drop net-telnet dependency; it's being removed from Ruby 2.3 for a reason.

--- .metadata.orig	Thu Jul  2 13:21:08 2015
+++ .metadata	Thu Jul  2 13:21:18 2015
@@ -39,20 +39,6 @@ dependencies:
       - !ruby/object:Gem::Version
         version: '0'
 - !ruby/object:Gem::Dependency
-  name: net-telnet
-  requirement: !ruby/object:Gem::Requirement
-    requirements:
-    - - ">="
-      - !ruby/object:Gem::Version
-        version: '0'
-  type: :runtime
-  prerelease: false
-  version_requirements: !ruby/object:Gem::Requirement
-    requirements:
-    - - ">="
-      - !ruby/object:Gem::Version
-        version: '0'
-- !ruby/object:Gem::Dependency
   name: bundler
   requirement: !ruby/object:Gem::Requirement
     requirements:
